main.go

```diff
@@ -216,6 +222,29 @@ func notifyWaiters(w http.ResponseWriter, r *http.Request) {
 
 func main() {
 	flag.Parse()
+	if os.Getenv("WASMSERVE") == "cp" {
```

Comment:
Hmm, can we make a separate program?

(I tried `-exec="go run cp.go"` but failed since `go run` is executed under `GOOS=js` in our case.)

Reply:
I'm not sure if we can make it a separate program, unless we do a separate `go build` and then invoke the resulting binary

Reply:
OK let's keep this 'embedding' way. Running `go build` for `cp.go` might be another idea, but we would have to consider more things for this...